references/gcp-sa-auth-shared-flow/sharedflowbundle/policies/RF.InvalidConfig.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+ Copyright 2020 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<RaiseFault name="RF.InvalidConfig">
+ <IgnoreUnresolvedVariables>true</IgnoreUnresolvedVariables>
+ <FaultResponse>
+   <Set>
+     <StatusCode>500</StatusCode>
+     <ReasonPhrase>Invalid Config</ReasonPhrase>
```

Comment:
Reason phrases should stick to standards - "Internal Server Error". I am happy for this message to be in the payload though